yt/frontends/gamer/data_structures.py

```diff
@@ -367,6 +367,8 @@ def _parse_parameter_file(self):
         else:
             self.mhd = 0
             self.srhd = 0
+            # set dummy value of mu here to avoid more complicated workarounds later
+            self.mu = 1.0
```

Comment:
Replace `self.parameters.setdefault("Opt__Unit", 0)` on line 374 with `self.unit = parameters.get("Opt__Unit", 0)`. Also check the related comments in `fields.py`.

Reply:
Done.